readthedocs/search/faceted_search.py

```diff
@@ -159,10 +165,12 @@ class PageSearchBase(RTDFacetedSearch):
     doc_types = [PageDocument]
     index = PageDocument._doc_type.index
 
-    _outer_fields = ['title^2']
-    _section_fields = ['sections.title^3', 'sections.content']
+    # boosting for these fields need to be close enough
+    # to be re-boosted by the page rank.
+    _outer_fields = ['title^1.5']
+    _section_fields = ['sections.title^2', 'sections.content']
```

Comment:
while one is greater than the other results remain the same, and we don't have any other factor that alter the boosting, so this doesn't change our current score.